lib/install/config/.babelrc

```diff
@@ -1,14 +1,24 @@
 {
   "presets": [
-    ["env", {
-      "modules": false,
-      "useBuiltIns": true
-    }]
+    [
+      "@babel/env",
+      {
+        "modules": false,
+        "targets": {
+          "forceAllTransforms": true
```

Comment:
`forceAllTransforms` is unneeded because we're targeting uglify-es (it's needed for old uglify without es6 support).

Reply:
Yeah I know, we sort of need it because the default target for Uglify is ES5 as well.  https://github.com/rails/webpacker/blob/master/package/environments/production.js#L29